### DISCUSSION TITLE
discuss: connection refused: /127.0.0.1:11800

### DISCUSSION THREAD

**Discussion by yijieshusheng-git** [apache/skywalking#10082] [Q&A] [answered]
my skywalking version is 8.7.0,and i only config backend_service ,the log will write "connection refused: /127.0.0.1:11800",
according to this dicussion https://github.com/apache/skywalking/discussions/8058
when i add these on agent.config:
plugin.toolkit.log.grpc.reporter.server_host=${SW_GRPC_LOG_SERVER_HOST:your oap ip address}
plugin.toolkit.log.grpc.reporter.server_port=${SW_GRPC_LOG_SERVER_PORT:11800}
plugin.toolkit.log.grpc.reporter.max_message_size=${SW_GRPC_LOG_MAX_MESSAGE_SIZE:10485760}
plugin.toolkit.log.grpc.reporter.upstream_timeout=${SW_GRPC_LOG_GRPC_UPSTREAM_TIMEOUT:30}
it is works!
but,how can config multiple addresses?

**Comment by kezhenxu94** [comment, marked as answer]
8.7.x is end of live. Multiple addresses are usually spectated by `,`

**Reply by yijieshusheng-git** [reply]
I have tried to do this, such as 
plugin.toolkit.log.grpc.reporter.server_host=${SW_GRPC_LOG_SERVER_HOST:ip1,ip2,ip3}
plugin.toolkit.log.grpc.reporter.server_port=${SW_GRPC_LOG_SERVER_PORT:11800}
plugin.toolkit.log.grpc.reporter.max_message_size=${SW_GRPC_LOG_MAX_MESSAGE_SIZE:10485760}
plugin.toolkit.log.grpc.reporter.upstream_timeout=${SW_GRPC_LOG_GRPC_UPSTREAM_TIMEOUT:30}
But the report was wrong。Is that what you mean？

**Reply by wu-sheng** [reply]
`plugin.toolkit.log.grpc.reporter.server_host` doesn't support this at all. And this had been removed for months.

**Reply by yijieshusheng-git** [reply]
OK，as 8.7.0 version,How can I set the addresses to take effect?

**Reply by wu-sheng** [reply]
No, you can't.

**Reply by yijieshusheng-git** [reply]
but at present，only config backend_service=ip1,ip2,ip3 it is not work
